src/blog/understanding-ecmascript-part-1.md

```diff
@@ -6,9 +6,13 @@ avatars:
 date: 2020-02-03 13:33:37
 tags:
   - ECMAScript
+  - Understanding ECMAScript
```

Comment:
Huh, do tags with spaces actually work and render fine?

Reply:
They seem to "work" as in they will get transformed to the space-free version. I can also use that explicitly if you'd prefer.

Reply:
Ok this comment was inaccurate. So, they render as UnderstandingECMAScript and the link is /blog/tags/understanding-ecmascript

So maybe it's fine? I'd like the link to be understanding-ecmascript and not understandingecmascript.

Reply:
Yeah the link sounds good. The rendering without spaces is a bit weird, but if you're comfortable with it, fine by me (I suppose we can fix it later if it's a problem).